.github/workflows/main.yml

```diff
@@ -41,6 +41,14 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Check free disk
+        run:  df -h
+      - name: Remove dotnet to free disk
+        run:  sudo apt purge dotnet*
+      - name: Remove android to free disk
+        run:  sudo rm -rf /usr/local/lib/android
+      - name: Check free disk
+        run:  df -h
```

Comment:
How about `dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | tail` and then selection the largest unused packages?

Reply:
I will try but there stuff on it like the android sdk which will not be listed :-(.

Without these modifications we have around 31G of free disk space. The SWUpdate example build needs < 35G.
Dotnet and android are 16G of Disk space.

Reply:
I see. And I wonder if there is anything under `/usr/local` for us then. Moby gets installed properly as package.

Reply:
I will only remove the /usr/local content.